public/video-ui/src/components/FormFields/DatePicker.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import Picker from 'react-datepicker';
+import DatePicker from 'react-datepicker';
 import moment from 'moment';
 import 'react-datepicker/dist/react-datepicker.css';
 import Icon from '../Icon';
 
-const DATE_FORMAT = 'DD MMM YYYY';
-const DATETIME_FORMAT = `${DATE_FORMAT} HH:mm`;
+const DATE_FORMAT = 'dd MMM yyyy';
+const DATETIME_FORMAT = `DD MMM yyyy HH:mm`;
 
```

Comment:
Upgrading `react-datepicker` meant we had to format the dates a bit differently to achieve the same result as the current build.